templates/model-driven-operations/index.html

```diff
@@ -0,0 +1,473 @@
+{% extends "model-driven-operations/_base_model.html" %}
+
+{% block title %}Model-driven operations: the best way to run enterprise software{% endblock %}
+
+{% block meta_description %}Save effort and regain focus through model-driven operations by declaratively composing your applications out of reusable, proven and flexible operators.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1G4sSrM-FQIL2c1Uj8PvW6oGUKcJ9HAK5TwnKhFa4sOY/edit#heading=h.kpn77t6yr8it{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h1>Model-driven operations: the best way to run enterprise software</h1>
+      <p>Modern software is complex and expensive to run reliably at scale. The setups are complicated and hard to maintain. Time to put structure in the chaos!</p>
+      <p>Model-driven operations let you save effort and focus by declaratively composing your applications out of reusable, proven and flexible operators and running them smoothly and toil-free.</p>
+    </div>
+    <div class="col-5 u-hide--small u-align--center u-vertically-center">
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/2c7e3fab-juju-header-illustration.svg",
+        alt="Model-driven operations with Juju",
+        width="396",
+        height="360",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
```

Comment:
![image](https://user-images.githubusercontent.com/57550290/146173045-cf452328-4b12-403b-91bf-71c64e9cb641.png)

Would it be better if these 3 sections have `col-8` as they have no images on the side? What do you think @spencerbygraves?